src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Navbar.scss';
+
+function Navbar() {
+  return (
+    <nav>
+      <div className="logo-container">
+        <h2>Math Magicians</h2>
+      </div>
+      <ul className="links-container">
+        {[{
+          id: 1,
+          path: '/',
+          text: 'Home',
+        },
+        {
+          id: 2,
+          path: '/calculator',
+          text: 'Calculator',
```

Comment:
- [optional] Great job. You can add the size of the calculator and as well add some more design or even background design to make it prettier and appealing. Nice work